### PR TITLE
Add property gist:description. Fixes #425.

### LINKED ISSUE #425: Add gist:description
We have many places where we'd like to add a user description field on various instance data and there doesn't seem to be a good existing choice for this. It seems general enough that it could be a candidate to go into gist.

### PR DESCRIPTION
**Pull request by rjyounes**
Add datatype property ```gist:description```.

### CODE REVIEW

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 =====
 
+Release X.x.x [ADD APPROPRIATE VERSION NUMBER FOR RELEASE]
+-----
+
+### Minor Updates
+
+- Added datatype property `gist:description` for describing instance data. Issue [#425](https://github.com/semanticarts/gist/issues/425).
+
+Import URL: <https://ontologies.semanticarts.com/o/gistCoreX.x.x>. [ADD APPROPRIATE VERSION NUMBER FOR RELEASE]
```
Comment (uscholdm) [inline docs/ReleaseNotes.md:11]: Why is this here? There is non corresponding change in any files.

File: docs/ReleaseNotes.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
 gist Release Notes
 =====
 
+Release X.x.x [ADD APPROPRIATE VERSION NUMBER FOR RELEASE]
+-----
+
+### Minor Updates
+
+- Added datatype property `gist:description` for describing instance data. Issue [#425](https://github.com/semanticarts/gist/issues/425).
```
Comment (uscholdm) [inline docs/ReleaseNotes.md:9]: Here is says for instance data. There is no mention of this in the actual ontology. Hard to know what is intended. Maybe add a scope note about intended usage?

**Review by uscholdm — CHANGES_REQUESTED** [review]
Clarify meaning of `gist:description`.
Remove or explain redundant text in release notes.

File: docs/ReleaseNotes.md (hunk 1 above)
Comment (rjyounes) [reply docs/ReleaseNotes.md:11]: This is the URL we provide for users to import gist. The bracketed note is addressed to the release manager and will be deleted in the final text.

**Review by sa-bpelakh — APPROVED** [review]
Perhaps a `skos:scopeNote` on the property to indicate that it is a description of an instance entity providing greater detail than a `prefLabel`? Otherwise, looks fine.

File: docs/ReleaseNotes.md (hunk 2 above)
Comment (rjyounes) [reply docs/ReleaseNotes.md:9]: Good idea, I added a scope note.

**Pull request merged**